native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -4007,6 +4020,14 @@ static const command_rec  proxy_cluster_cmds[] =
         OR_ALL,
         "UseNocanon - When no ProxyPass or ProxyMatch for the URL, passes the URL path \"raw\" to the backend (Default: Off)"
     ),
+
+    AP_INIT_FLAG(
+        "Return503WhenNoContext",
```

Comment:
Maybe `"NoContext503"` would be less wordy?

Reply:
+1 I think we need better name for this as 'Return503WhenNoContext' is very cumbersome.